alacritty/src/config/mod.rs

```diff
@@ -230,7 +230,8 @@ pub fn deserialize_config(path: &Path) -> Result<Value> {
             "YAML config {path:?} is deprecated, please migrate to TOML using `alacritty migrate`"
         );
 
-        let value: serde_yaml::Value = serde_yaml::from_str(&contents)?;
+        let mut value: serde_yaml::Value = serde_yaml::from_str(&contents)?;
+        prune_yaml_nulls(&mut value);
```

Comment:
Won't this also remove nulls which are recognized as error already?

Reply:
This will only remove unit types, I'm not sure you can craft `nul` as error.

Reply:
If you do `window:` it will have a unit type and be an error.

Reply:
But do we care? Like removing it during translation is better than forcing users to figure out what to actually fix, because there's no clear indication on what is being broken, and they could have an error before for that, no?

Like I don't really want to write code based on broken config that throws an error before, I write it based on the configs that worked before but not anymore.